### PR TITLE
Feature/UAR 695 change links for review update statement page

### PR DESCRIPTION
**Pull request by bmcgreevyCH**
### JIRA link
https://companieshouse.atlassian.net/browse/UAR-695

### Change description
Change links for review update statement page.

### CODE REVIEW

File: src/utils/navigation.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ export const NAVIGATION: Navigation = {
   [config.UPDATE_REVIEW_STATEMENT_URL]: {
     currentPage: config.UPDATE_REVIEW_STATEMENT_PAGE,
     previousPage: () => config.UPDATE_NO_CHANGE_REGISTRABLE_BENEFICIAL_OWNER_URL,
-    nextPage: []
+    nextPage: [config.WHO_IS_MAKING_UPDATE_PAGE, config.UPDATE_NO_CHANGE_BENEFICIAL_OWNER_STATEMENTS_PAGE]
```
Comment (bbell-companieshouse) [inline src/utils/navigation.ts:65]: Should we add a 'todo' here referencing that UAR 585 will change this to go to payments page?

Comment (bmcgreevyCH) [reply src/utils/navigation.ts]: That was a hangover from a merge. Changed. Cheers,

File: test/controllers/update/update.review.statement.controller.spec.ts [hunk 2]
```diff
@@ -84,8 +84,6 @@ describe("Update review overseas entity information controller tests", () => {
       expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_WHO_CAN_WE_CONTACT);
       expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_BENEFICIAL_OWNER_STATEMENT);
       expect(resp.text).toContain(NO_CHANGE_REVIEW_STATEMENT_BENEFICIAL_OWNER_STATEMENTS_CEASED_TITLE);
-      expect(resp.text).toContain(UPDATE_PRESENTER_CHANGE_FULL_NAME);
```
[the remark this answers is not in the document]
Comment (bmcgreevyCH) [reply test/controllers/update/update.review.statement.controller.spec.ts:87]: They were moved to another test below where all change links were grouped together. Just thought it was neater.

File: views/update/review-update-statement.html [hunk 3]
```diff
@@ -38,7 +38,7 @@
 {% endset %}
 
 {% set beneficialOwerSectionHeading %}
-  {% if not pageParams.noChangeFlag %}
+  {% if not pageParams.isNoChangeJourney %}
```
[the remark this answers is not in the document]
Comment (bmcgreevyCH) [reply views/update/review-update-statement.html:41]: Merge again. Corrected. Thank you.

**Review by n-r-b — APPROVED** [review]
Looks good, just a very minor comment

File: test/controllers/.DS_Store
Comment (n-r-b) [inline test/controllers/.DS_Store:1]: Should this be checked in?

File: test/controllers/update/update.review.statement.controller.spec.ts [hunk 4]
```diff
@@ -131,6 +129,19 @@ describe("Update review overseas entity information controller tests", () => {
       expect(resp.text).toContain("Individual managing officer");
     });
 
+    test(`renders the ${UPDATE_REVIEW_STATEMENT_PAGE} page renders change links`, async () => {
```
Comment (AndreKramer7654) [inline test/controllers/update/update.review.statement.controller.spec.ts:132]: minor two renders in test name. "with change links"?

**Review by AndreKramer7654 — APPROVED** [review]

**Pull request merged**